lib/events/Events.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * An interface representing all the collection of all of the posts in the app.
+ */
+export interface Events {
+  /**
+   * Fetches events with the given ids.
+   *
+   * @returns a list of events (right now just mocks a bunch of events)
+   */
+  eventsWithIds: (ids: string[]) => Event[]
```

Comment:
Just a heads up that in the near future we'll probably need this to be something like `eventsInLocation` that takes in location coordinates and a radius.